core/src/main/java/io/dekorate/kubernetes/decorator/AddResourcesRequirementDecorator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 The original authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dekorate.kubernetes.decorator;
+
+import io.dekorate.kubernetes.annotation.Resources;
```

Comment:
Use `io.dekorate.kubernetes.config.Resources` instead of you have to.

Why?

The annotations are just meant to be used by the user and not internally by decorate. Internally the annotation is converted to a pojo (usually found under the ../config package).

An even better appraoch here, would be to just use raw values:

- int requestsCpu
- int requestsMem
- int limitCpu
- int limitMem

as this would allow the decorator to be used by downstream projects that use their own config model  (e.g. Quarkus).